mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -3717,8 +3731,12 @@ func expandMaintenancePolicy(d *schema.ResourceData, meta interface{}) *containe
 
 func expandClusterAutoscaling(configured interface{}, d *schema.ResourceData) *container.ClusterAutoscaling {
 	l, ok := configured.([]interface{})
+	enableAutopilot := false
+	if v, ok := d.GetOk("enable_autopilot"); ok && v == true {
+		enableAutopilot = true
+	}
 	if !ok || l == nil || len(l) == 0 || l[0] == nil {
-		if v, ok := d.GetOk("enable_autopilot"); ok && v == true {
+		if enableAutopilot {
```

Comment:
I am trying to understand the change here. Does this new change cover any cases that the previous code does not cover?

Reply:
It's not a logical change, however it makes additional autopilot conditionals simpler in the future